client/signup/steps/site-information/index.jsx

```diff
@@ -178,38 +179,36 @@ class SiteInformation extends Component {
 }
 
 export default connect(
-	state => {
+	( state, ownProps ) => {
 		const siteType = getSiteType( state );
+		const isBusiness = 'business' === siteType;
+		const formFields =
+			! isBusiness && 'site-information' === ownProps.stepName
```

Comment:
This removes the address and phone fields for the `onboarding` flow, however the `onboarding-dev` flow will still show the two fields. I may have misunderstood the plan, but is this because we want to wait for #29863?

Reply:
It does not remove them - all of the fields are present in the `onboarding` flow (with Business site type selected) thanks to the `informationFields` prop [defined here](https://github.com/Automattic/wp-calypso/blob/99fd7722a34553a86fc074a4cf00ff8e6441dd27/client/signup/config/steps-pure.js#L391)